Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -199,25 +199,49 @@ define([
         var byteLength = view.getUint32(byteOffset, true);
         byteOffset += sizeOfUint32;
 
+        var batchTableJsonByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var batchTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
         var batchLength = view.getUint32(byteOffset, true);
-        this._featuresLength = batchLength;
         byteOffset += sizeOfUint32;
 
+        // TODO : remove this legacy check before merging into master
```

Comment:
@lilleyse when this is merged, can you also email the forum about this small breaking change for b3dm?